whatsapp_business_api_is/utils.py

```diff
@@ -176,7 +176,8 @@ def set_data(user, data, msg):
         obj = run_action(data.get('action'), user, msg, None, data)
 
     if 'field' in data:
-        value = data.get('value', msg.validated_value)
+        # we want an exception to be thrown if msg is None and data['value'] doesn't exist
```

Comment:
I'm not sure I understand.
If value is not in data, and msg is None, an exception will be thrown.
You just replace one exception with another? Add try and catch, it's much more readable..